Backend/mentii/class_ctrl.py

```diff
@@ -27,6 +27,26 @@ def getActiveClassList(dynamoDBInstance, email=None):
     response.addToPayload('classes', classes)
   return response
 
+def getTaughtClassList(dynamoDBInstance, email=None):
+  response = ControllerResponse()
+  if email is None:
+    email = g.authenticatedUser['email']
+  usersTable = dbUtils.getTable('users', dynamoDBInstance)
+  classTable = dbUtils.getTable('classes', dynamoDBInstance)
+  if usersTable is None or classTable is None:
+    response.addError(  'Get Taught Class List Failed','Unable to access users and/or classes')
```

Comment:
The error is logged as a part of adding the error to the response. See ResponseCreation.py line 21.